test/splitter.js

```diff
@@ -153,5 +172,24 @@ contract('Splitter', async accounts => {
         checkEventNotEmitted();
     });
 
+
+
+    it("withdraw is pausable", async () => {
+
+        splitter.pause({from: contractOwner});
+
+        await truffleAssert.reverts(
+            splitter.withdraw(toBN(0), {from: recipientTwoAddress}),
```

Comment:
At the same time, trying to withdraw 0, we know this would fail, so it would be better to test with parameters that would pass if not for the paused state.